src/drivers/extjs/components/dataview.js

```diff
@@ -4,14 +4,14 @@ import {HTMLComponentBase} from '../../html/components/base.js'
 import {ExtJsComponentBase} from './base.js'
 
 export class ExtJsComponentDataView extends ExtJsComponentBase {
-    select (callback, index, itemCls) {
+    select (callback, index) {
         const cmp = this.extJsComponent
         let htmlElement = null
 
         try {
             htmlElement = document
                 .getElementById(cmp.el.id)
-                .getElementsByClassName(itemCls || 'x-dataview-item')[index]
+                .getElementsByClassName(cmp.itemCls)[index]
```

Comment:
Looks reasonable for me to use `cmp.itemCls || 'x-dataview-item'` in case `cmp.itemCls` was not being defined?